composer.json

```diff
@@ -14,6 +14,7 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=5.3.9",
+        "doctrine/orm": "~2.2,>=2.2.3,<2.5",
```

Comment:
Why not ``doctrine/orm 2.5``? Tip: ``~2.2,>2.2.3`` can be replaced with ``^2.2.3`` https://getcomposer.org/doc/articles/versions.md#caret.